docker-compose.yml

```
@@ -0,0 +1,36 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: ./docker/dev/Dockerfile
```

```suggestion
      dockerfile: ./Dockerfile
```
Having something like this and then have docker file at the root